releasenotes/notes/add-github-changelog-workflow-dfcda8709ddcf2d3.yaml

```diff
@@ -0,0 +1,5 @@
+---
+other:
+  - |
+    Add new GitHub workflow to validate that Reno release notes are added to every pull request,
```

Comment:
What format + sections do we want?

How do we make this include a link to the PR + author (for external contributors), how do we do that automatically (?)

Reply:
This is user-facing release notes. I don't think it must have links to PRs.

Reply:
We do that today in our changelog, keeping link to PR + outside contributor, would be great to keep that linkage + attribution if we can.

Reply:
We have it in `git log`.
The question is: what's the value of presenting this attribution and link to the user?

Reply:
I don't think this reno entry is actually necessary since the release notes are for users of the library. Should we remove it @brettlangdon?

Reply:
oh... probably?